src/components/Explore/queries/generateMetricsQuery.ts

```diff
@@ -11,7 +11,9 @@ export function generateMetricsQuery({ metric, groupByKey, extraFilters, groupBy
   // Generate span set filters
   let filters = `${VAR_FILTERS_EXPR}`;
 
-  if (metric === 'errors') {
+  if (metric === 'rate') {
+    filters += ' && status!=error';
```

Comment:
Hmm... 🤔  Does it mean we drop errors altogether in rate metric? Wouldn't it be confusing (e.g. if there are only errors the rate would be 0)?

| before | after |
|---|---|
| <img width="1022" alt="Screenshot 2025-03-26 at 13 24 27" src="https://github.com/user-attachments/assets/cf2073da-d427-4155-8264-1dcabe9e5c14" /> | <img width="1021" alt="Screenshot 2025-03-26 at 13 24 32" src="https://github.com/user-attachments/assets/906b8fdf-7890-4161-9923-8955089f26a2" /> |